tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -208,25 +459,79 @@ public function callMapEntryProvider(): iterable
         }
     }
 
+    /**
+     */
+    private function isIgnored(string $functionName): bool
+    {
+        /** @psalm-assert callable-string $functionName */
```

Comment:
Not sure what @psalm-assert inline does. Did you meant @var?

Reply:
Hmm I think that one can be removed. Older version of the code has a function exists check I think.